cmd/hello/main.go

```diff
@@ -14,10 +14,10 @@ func main() {
 
 	w := app.NewWindow("Hello")
 	w.SetContent(&widget.Box{Horizontal: false, Children: []fyne.CanvasObject{
-		&widget.Label{Text: "Hello Fyne!"},
-		&widget.Button{Text: "Quit", OnTapped: func() {
```

Comment:
This button line need not change, as the API for that was not modified.
If we change Text.buffer to Text.Text then the label line above could be maintained as well...

Reply:
Is there a reason to move to the function version of this? It makes the file inconsistent.
I don't have a preference for one over the other really but it should probably match the box constructor for consistency.

Reply:
Apologies it was probably lost during the merge, Added back in 9d50247c340f7de3a85b3e8285ee7f9ca1a6fbc9